Formula/e/ehco.rb

```diff
@@ -22,22 +23,21 @@ class Ehco < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "566974576744f5056e4a7d4f5d3b14b0b6a4af3f25a8751f5fa5c1cf901c6e83"
   end
 
-  depends_on "go@1.19" => :build
+  depends_on "go@1.20" => :build
 
   uses_from_macos "netcat" => :test
 
-  # Backport quic-go update to support Go 1.19
-  # Remove in the next release
+  # go@1.20 build patch, remove in next release
   patch do
-    url "https://github.com/Ehco1996/ehco/commit/2f739a4279f9defaeb8beac9e97e82851e0dd995.patch?full_index=1"
-    sha256 "43235566b344e0d125d79719800f1de61cd032f89892b183adf8220b1c5cd298"
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/cb97010/ehco/1.1.2-go-1.20-build.patch"
```

Comment:
```suggestion
    url "https://raw.githubusercontent.com/Homebrew/formula-patches/cb97010770c6678102b3302e904bb32c6169311c/ehco/1.1.2-go-1.20-build.patch"
```

Reply:
why do prefer the long hash?

Reply:
nb i did not change the hash, as it is the same thing :)

Reply:
When the repo has sufficiently long history, the chance of getting clashes in short hashes may no longer be negligible. Just wanted to wipe out any potential ambiguity. But ok, I'm fine with that too.